pinot-core/src/main/java/com/linkedin/pinot/core/indexsegment/generator/SegmentGeneratorConfig.java

```diff
@@ -15,6 +15,21 @@
  */
 package com.linkedin.pinot.core.indexsegment.generator;
 
+import com.google.common.base.Preconditions;
```

Comment:
Moving SegmentPartitionConfig from core to common now requires adding the following explicitly.
import com.linkedin.pinot.common.config.SegmentPartitionConfig

That is the change in this file.